### DISCUSSION TITLE
How does this library compare to `OpenSimplex2` (`SuperSimplex`)?

### DISCUSSION THREAD

**Discussion by tommy-mitchell** [jwagner/simplex-noise.js#63] [General]
I can't seem to find any comparisons between this library and https://github.com/KdotJPG/OpenSimplex2. Is `simplex-noise.js` based on the original `OpenSimplex`, or the Ken Perlin version?

**Comment by jwagner** [comment]
It's based on the original Simplex Noise, or more specifically, Simplex noise demystified by Stefan Gustavson.
